modules/custom/apigee_kickstart_customizer/src/Customizer.php

```diff
@@ -113,6 +147,31 @@ public function getDefinitionForTheme($theme = NULL): array {
     return isset($definitions[$theme]) ? $definitions[$theme] : [];
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function updateStylesheetForTheme($theme = NULL): void {
+    $theme = $theme ?? $this->activeTheme->getName();
+    $values = $this->configFactory->get('apigee_kickstart_customizer.theme.' . $theme)
+      ->get('values');
+
+    // TODO: Add scope support.
+    $css = ':root {';
+    foreach ($values as $name => $value) {
```

Comment:
I'm getting an error here when I try to delete this config.  Should be able to delete this, and stop the file write, etc.

Reply:
Pushed a fix for it. On config delete, the stylesheet is deleted.

Reply:
Yep, works perfectly!  Thanks @arshad :)